src/components/MoviesList/MoviesList.js

```diff
@@ -4,15 +4,23 @@ import PropTypes from 'prop-types';
 import './MoviesList.scss';
 import { MovieCard } from '../MovieCard';
 
-export const MoviesList = ({ movies }) => (
-  <div className="movies">
-    {movies.map(movie => (
-      <MovieCard key={movie.imdbId} {...movie} />
-    ))}
-  </div>
-);
+export const MoviesList = ({ movies, query }) => {
+  const visibleMovies = movies.filter(
+    movie => movie.title.toLowerCase().includes(query.toLowerCase())
+      || movie.description.toLowerCase().includes(query.toLowerCase()),
```

Comment:
(title + description).toLowerCase().includes(...)

Reply:
wow, didn't know about this possibility👍